test_runner/logical_repl/test_log_repl.py

```diff
@@ -34,21 +37,68 @@ def query_clickhouse(
     raise ValueError("Hash mismatch")
 
 
+class DebeziumAPI:
+    """
+    The class for Debeziym API calls
+    """
+
+    def __init__(self, base_url: str):
+        self.__base_url = base_url
+        self.__connectors_url = self.__base_url + "/connectors"
```

Comment:
Let's use f-strings for strings concatenation here and in other places

```suggestion
        self.__connectors_url = "#{self.__base_url}/connectors"
```

Reply:
Done, thanks